### PR TITLE
Enhancement/add force parameters to infrasim init

### PR DESCRIPTION
**Pull request by chenge3**
Add '-f' parameter for "Infrasim init"
If there is node workspace existing, "init" action will be broken without '-f' parameter.
With '-f' parameter, the node will be destroy first, then init action.

### CODE REVIEW

**Comment by chenge3** [comment]
@turtle-fly @XiaowenJiang @JuneZhou19 @WinnieXU1 @xiar @fub2 
Comments are always welcomed. 
Thanks.

**Comment by XiaowenJiang** [comment]
Tried it. +1

**Comment by fub2** [comment]
can you supply with functional test?

File: test/functional/test_cli_commands.py [hunk 1]
```diff
@@ -265,6 +265,22 @@ def test_start_info_destroy_info(self):
...
+        output_info = {}
+        output_start = run_command("infrasim node start")
+        self.assertEqual(output_start[0], 0)
+        self.assertTrue(Workspace.check_workspace_exists(self.node_name))
+        output_info['start'] = run_command("infrasim node info")
+        self.assertEqual(output_info['start'][0], 0)
+
+        output_init = run_command("infrasim init -s")
+        self.assertEqual(output_init[0], 0)
+
```
Comment (fub2) [inline test/functional/test_cli_commands.py]: is it necessary to run check w/ "self.assertTrue(Workspace.check_workspace_exists(self.node_name))"?

Comment (chenge3) [inline test/functional/test_cli_commands.py]: The work space will be created after "infrasim node start", I will add a infrasim_home check.

**Pull request merged**